content/blog/faster-secrets-management/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Faster Stack Secrets in Pulumi IaC"
```

Comment:
Any reason not to just say Faster Secrets? We don’t typically call them “Stack Secrets”.

Reply:
Trying to not confuse people given we've also got a separate product called "Secrets" within "ESC" .. not sure how else to highlight that this isn't about ESC Secrets.